test/gas-estimation/InterchainGasEstimation.js

```diff
@@ -80,4 +81,20 @@ describe('InterchainGasEstimation', () => {
 
         expect(estimate).to.equal(212504600090000);
     });
+
+    it('should compute gas estimate for Scroll', async () => {
+        await gasEstimate
+            .updateGasInfo(destinationChain, [ScrollEstimationType, 90000, 190000, 5000, 0])
+            .then((tx) => tx.wait());
+
+        const estimate = await gasEstimate.estimateGasFee(
+            destinationChain,
+            destinationAddress,
+            '0x2534d1533c9ffce84d3174c1f846a4041d07b56d1e7b5cb0000000000007138e06fb42086325',
+            120000,
+            '0x',
+        );
+
+        expect(estimate).to.equal('6552710823071820600090000');
```

Comment:
estimate is very off. This is about 6M ETH. Seems to be because you're essentially multiplying L1 base fee twice.

Reply:
You were right